firmware/integration/rusefi_config.txt

```diff
@@ -567,10 +572,10 @@ log_format_e logFormat;
 	int HD44780width;;"index", 1, 0, 0, 300, 0
 	int HD44780height;;"index", 1, 0, 0, 300, 0
 
-	adc_channel_e tps1_1AdcChannel;First throttle body, first sensor. See also pedalPositionAdcChannel
+	adc_channel_e tps1_1AdcChannel;+First throttle body, first sensor. See also pedalPositionAdcChannel
 adc_channel_e vbattAdcChannel;+This is the processor input pin that the battery voltage circuit is connected to, if you are unsure of what pin to use, check the schematic that corresponds to your PCB. 
 adc_channel_e fuelLevelSensor;+This is the processor pin that your fuel level sensor in connected to. This is a non standard input so will need to be user defined. 
-	adc_channel_e tps2_1AdcChannel;Second throttle body position sensor, single channel so far\nset_analog_input_pin tps2 X
+	adc_channel_e tps2_1AdcChannel;+Second throttle body position sensor, single channel so far\nset_analog_input_pin tps2 X
```

Comment:
`second throttle, primary sensor`
no need for console command here

Reply:
set_analog_input_pin tps2 X ?